testsuite/do_tests.sh.in

```diff
@@ -412,6 +420,8 @@ echo >  "${TEST_LOGFILE}" "NEST v. @NEST_VERSION_VERSION@ testsuite log"
 echo >> "${TEST_LOGFILE}" "======================"
 echo >> "${TEST_LOGFILE}" "Running tests from ${TEST_BASEDIR}"
 
+CODES_SKIPPED=' 200 Skipped'
```

Comment:
I'm not sure about that but one may ask why these tests are skipped. `CODES_SKIPPED=' 200 Skipped (build with-mpi=OFF required)'` or something like that would make it clear. We have similar comments at other places, e.g. `Success (expected failure)`.